test/e2e/operator_test.go

```diff
@@ -0,0 +1,100 @@
+// +build e2e
```

Comment:
need to add copyright header (FYI, we recently added a lint check for this in Contour, would be good to do the same here: https://github.com/projectcontour/contour/pull/3133)

Reply:
@skriss commit `644695b` resolves this comment. I created https://github.com/projectcontour/contour-operator/issues/140 to address the boilerplate check.